rest-api/responses/index.md

```diff
@@ -0,0 +1,211 @@
+# Responses
```

Comment:
The slug is `responses-2` on [the handbook page](https://developer.wordpress.org/plugins/rest-api/responses-2/), probably because another page already uses the slug `responses`.

For this PR, I removed the number.

Reply:
In this case, please use the same name, so that could be synced, this file would be under the directory `responses-2`